.github/workflows/publish-npm.yml

```diff
@@ -88,16 +88,9 @@ jobs:
         run: |
           npm publish
 
-      - uses: montudor/action-zip@v1
-        if: ${{ inputs.github_release == 'true' }}
+      - uses: uniquesca/continuous-integration/publish-github-release@feature/publish-github-release
```

Comment:
You still need to use `if: ${{ inputs.github_release == 'true' }}` here, because in some cases we just want to publish npm package. Github release is like a nice optional addition here.